### ISSUE TITLE
Απορια για το 1.6

### ISSUE THREAD

**Issue by mpektkd** [ntua-cvsp-lab/DSP_LabSupport#36]
Επισης ηθελα να ρωτησω.Η συναρτηση θα ειναι για σηματα τα οποια θα ξεκινανε με 1000 δειγματα 100 μηδενικα κλπ...γιατι αυτο καταλαβαινω απο την εκφωνηση που λεει για σηματα οπως του ερωτ.3 .Ευχαριστω

**Comment by cgaroufis** [comment]
Όχι, η συνάρτηση θα πρέπει να λειτουργεί στη γενικότερη περίπτωση όπου ναι μεν οι τόνοι θα έχουν μήκος 1000 δείγματα, αλλά δε θα ξέρεις το μεσοδιάστημα ανάμεσα σε 2 διαδοχικούς τόνους. Μπορείς να τσεκάρεις την απάντηση [εδώ](https://github.com/ntua-cvsp-lab/DSP_LabSupport/issues/12) για τρόπους αντιμετώπισης του παραπάνω προβλήματος.

Χρήστος
